db/query.js

```diff
@@ -29,10 +28,17 @@ export default {
             console.error(err);
         })
     },
+    /* finds the next open room
+    for a FULL room, if user reloads, they will either be
+    placed into a new room or join the oldest non-full room
+    
+    otherwise, for normal reload, user will be placed into the
+    oldest non-full room
+    */
 
-    findOpenSession() {
+    findOpenSession(size) {
         return client.query(`
-        SELECT sessionId FROM sessions WHERE numConnected <= 8
+        SELECT sessionId FROM sessions WHERE numConnected < ` + size + `
```

Comment:
Is `size` sanitized?

Reply:
Ah I see that it's based on ROOM_SIZE which is a value that you're setting. Please ignore.